app/src/main/java/com/hedvig/app/feature/trustly/TrustlyConnectFragment.kt

```diff
@@ -107,14 +108,16 @@ class TrustlyConnectFragment : Fragment(R.layout.trustly_connect_fragment) {
                 webViewClient = object : WebViewClient() {
                     override fun onPageFinished(view: WebView?, url: String?) {
                         super.onPageFinished(view, url)
-                        if (!hasLoadedWebView) {
-                            TransitionManager.beginDelayedTransition(
-                                binding.root,
-                                MaterialFadeThrough()
-                            )
-                            loadingContainer.isVisible = false
-                            trustly.isVisible = true
-                            hasLoadedWebView = true
+                        if (viewLifecycleOwner.lifecycle.currentState.isAtLeast(Lifecycle.State.CREATED)) {
```

Comment:
Changed to doing this synchronously

Reply:
Looks good! We can simplify the if-statement to one-level probably, but this is good enough to merge 😄